examples/csi-config-map-sample.yaml

```diff
@@ -12,6 +12,12 @@ kind: ConfigMap
 # each such cluster in use.
 # To add more clusters or edit MON addresses in an existing configmap, use
 # the `kubectl replace` command.
+# The <rados-namespace> is optional and represents a radosNamespace in the pool.
+# If any given all the of the rbd images, snapshots and other metadata
```

Comment:
to me, it feels like we need to reword this and below line @mehdy

Reply:
Sure! How about this?

"To use a specific rados namespace for your rbd images and snapshots, provide the name of the desired rados namespace".

If you have something else in mind, I'd be glad to use the help.

Reply:
@mehdy it looks good to me ! :+1: